setup.py

```diff
@@ -1,8 +1,8 @@
 from setuptools import setup
 
 setup(
-    name='sparktestingbase',
-    version='0.0.7-snapshot',
+    name='spark-testing-base',
```

Comment:
Why are we changing this package's name?

Reply:
fixed